apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -245,6 +249,25 @@ schema("/plugins/:name/move") ->
                 400 => emqx_dashboard_swagger:error_codes(['MOVE_FAILED'], <<"Move failed">>)
             }
         }
+    };
+schema("/plugins/cluster_sync") ->
+    #{
+        'operationId' => sync_plugin,
+        post => #{
+            summary => <<"Sync specific version of plugin from given node">>,
```

Comment:
Maybe "from current node"? Seems there is no node-like parameter specified.

Reply:
nice catch. it's my bad.
The previous design required a specified node name, but this parameter was later removed.